src/app/main.js

```diff
@@ -256,8 +220,23 @@ define([
   };
 
   // finally, start up the app!
+  //init Map;
+  app.map = BootstrapMap.create('map', mapUtils.config.mapOptions);
```

Comment:
Move this into a new function in map.js called `createMap` that takes the DOM node id as an argument.